.github/dependabot.yml

```diff
@@ -10,3 +10,15 @@ updates:
       allow:
           - dependency-name: "@khanacademy/eslint-config"
           - dependency-name: "@khanacademy/eslint-plugin"
+      assignees:
+          - "@Khan/perseus"
+
+      # Grouped updates for Wonder Blocks and Wonder Stuff releases.
+      # This helps us to stay in sync with the latest releases of these packages.
+      groups:
+          wonder-stuff:
+              patterns:
+                  - "@khanacademy/wonder-stuff-*"
+          wonder-blocks:
+              patterns:
+                  - "@khanacademy/wonder-blocks-*"
```

Comment:
Can we filter out certain packages? I'm asking as we have a couple of WB deprecated packages that we usually skip when upgrading webapp (`wb-color` and `wb-spacing`).

It would be nice if we could skip them programmatically with dependabot so we never see these packages in here, as it would be noisy/counterproductive to have them included.

Reply:
We don't have any usages of `wb-color` nor `wb-spacing` in Perseus today. I can exclude them here, but I think I'm going to add an ESLint rule to ensure it doesn't come back. :) 

<img width="797" alt="image" src="https://github.com/user-attachments/assets/308b6fa3-e7a0-4c17-adf6-a3ab6afd5891">

```json
    "no-restricted-imports": [
        "error",
        "@khanacademy/wonder-blocks-color",
        "@khanacademy/wonder-blocks-spacing",
    ],
```